include/ocpp/v201/charge_point.hpp

```diff
@@ -133,6 +133,24 @@ class ChargePoint : ocpp::ChargingStationBase {
     int network_configuration_priority;
     bool disable_automatic_websocket_reconnects;
 
+    // store the connector status
+    struct evse_connector_pair {
```

Comment:
could also be defined publicly, I could imagine we could reuse this for other purposes. We could also move this later though